packages/runtime/src/next/api-handler/handlers/proxy-preview-mode.ts

```diff
@@ -60,7 +64,8 @@ async function proxyPreviewModeApiRouteHandler(
   res: NextApiResponse<ProxyPreviewModeResponse>,
   { apiKey }: { apiKey: string },
 ): Promise<void> {
-  const previewModeProxy = createProxyServer({ xfwd: true })
```

Comment:
Are we adding `X-Forwarded-*` headers?

Reply:
Confirmed we are
![Screenshot 2024-03-21 at 17 42 31](https://github.com/makeswift/makeswift/assets/10539418/18ed9a24-7ddc-4c06-8311-f44f1f2de034)